Makefile

```diff
@@ -28,9 +29,9 @@ install:
 	# ensure resolving works inside the chroot
 	cat /etc/resolv.conf > $(DESTDIR)/etc/resolv.conf
 	# copy-in launchpad's build archive
-	if grep -q ftpmaster.internal /etc/apt/sources.list; then \
+	if grep -q ftpmaster.internal /etc/apt/sources.list.d/lp-buildd.sources; then \
 		cp /etc/apt/sources.list $(DESTDIR)/etc/apt/sources.list; \
-		cp /etc/apt/trusted.gpg $(DESTDIR)/etc/apt/ || true; \
+		cp /etc/apt/sources.list.d/lp-buildd.sources $(DESTDIR)/etc/apt/sources.list.d/lp-buildd.sources; \
```

Comment:
does this need to be conditional?

Reply:
should probably match the existing pattern yes